taskcat/stacker.py

```diff
@@ -258,6 +258,35 @@ def set_parameter_path(self, parameter):
     def get_parameter_path(self):
         return self.parameter_path
 
+    def get_param_includes(self, s_params):
+        # Github/issue/57
+        # Fetch overrides from S3.
+        param_path = self.parameter_path
+        _url_prefix=re.search(r'^http.*/.*/(?!\/.*?$)', param_path).group[:-1]
+        override_url = "{}/{}".format(_url_prefix, 'override.json')
+        r = requests.get(override_url)
```

Comment:
To match the changes in development can you make this an s3 obj get call instead. This is necessary to support private buckets